tcf_website/templates/course/course.html

```diff
@@ -51,7 +51,7 @@ <h4 class="card-title">
                 {% for i in instructors %}
                     <li class="instructor {% if i.semester_last_taught != latest_semester %}old{% endif %}">
                         {% url 'course_instructor' course_id=course.pk instructor_id=i.pk as link %}
-                        {% include "common/rating_card.html" with title=i.full_name link=link rating=i.rating difficulty=i.difficulty gpa=i.gpa last_taught=i.semester_last_taught %}
+                        {% include "common/rating_card.html" with title=i.full_name link=link rating=i.rating difficulty=i.difficulty gpa=i.gpa last_taught=i.semester_last_taught current_semester=latest_semester %}
```

Comment:
Seems like this wasn't deleted from past commits?

Reply:
I went ahead and removed it, not sure if that's what you meant

Reply:
It was! It's not supposed to be there, right?

Reply:
Ahh yeah you're right it was from a previous edit that I deleted. Thanks for the catch!